airbyte-integrations/connectors/source-surveymonkey/integration_tests/abnormal_state.json

```diff
@@ -9,22 +9,243 @@
   {
     "type": "STREAM",
     "stream": {
-      "stream_state": {
```

Comment:
I think we should add `breaking_change` label to this PR, as stream_state format was changed.

Reply:
Updated